pkg/canary/reader/reader.go

```diff
@@ -207,7 +210,7 @@ func (r *Reader) closeAndReconnect() {
 		c, _, err := websocket.DefaultDialer.Dial(u.String(), r.header)
 		if err != nil {
 			fmt.Fprintf(r.w, "failed to connect to %s with err %s\n", u.String(), err)
```

Comment:
As you did above, perhaps you want to add how many seconds it will sleep for in the log message?